### PR TITLE
Line endings and CS0067 warnings fixed

### PR DESCRIPTION
**Pull request by olmisa**
* [x] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the Windows code?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

CS0067 disabled for CrashesDelegate events. Line endings fixed.

## Misc

[AB#71180](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/71180)

### CODE REVIEW

File: Assets/AppCenter/Plugins/AppCenterSDK/Distribute/Shared/UpdateAction.cs [hunk 1]
```diff
@@ -11,8 +11,8 @@ public enum UpdateAction
         /// <summary>
         /// Action to trigger the download of the release.
         /// </summary>
-        Update,
-
+        Update,
```
Comment (dhei) [inline Assets/AppCenter/Plugins/AppCenterSDK/Distribute/Shared/UpdateAction.cs:14]: I am curious, what is the end-of-line change here? If there's an inconsistency between CRLF (Windows) and LF (macOS), yes we should fix it.

On a mac, here is what I saw in `git diff`:
image

Comment (olmisa) [reply Assets/AppCenter/Plugins/AppCenterSDK/Distribute/Shared/UpdateAction.cs:14]: Yes, there was an inconsistency between CRLF (Windows) and LF (macOS). Unity shows a warning about it.

**Review by annakocheshkova — DISMISSED** [review]

**Comment by guperrot** [comment]
Please update changelog as these warnings are visible in editor when consuming the SDK.

**Review by thyeggman — DISMISSED** [review]

**Review by guperrot — DISMISSED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -18,6 +18,8 @@ Updated native SDK versions:
 
 ### App Center Crashes
 
+* **[Fix]** Fix Unity warnings in CrashesDelegate class.
```
Comment (MatkovIvan) [inline CHANGELOG.md]:
```suggestion
* **[Fix]** Fix Unity warnings in `CrashesDelegate` class.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -30,6 +32,8 @@ Updated native SDK versions:
 
 ### App Center Distribute
 
+* **[Fix]** Fix line endings Unity warning in UpdateAtion class.
```
Comment (MatkovIvan) [inline CHANGELOG.md]:
```suggestion
* **[Fix]** Fix line endings Unity warning in `UpdateAction` class.
```

**Review by MatkovIvan — CHANGES_REQUESTED** [review]

**Review by MatkovIvan — APPROVED** [review]

**Review by guperrot — APPROVED** [review]

**Review by dhei — APPROVED** [review]

**Pull request merged**